Description: The define for memory management gets toggled by the build
Author: Dirk Eddelbuettel <edd@debian.org>

--- aws-sdk-cpp-only-s3-1.7.275.orig/aws-cpp-sdk-core/include/aws/core/SDKConfig.h
+++ aws-sdk-cpp-only-s3-1.7.275/aws-cpp-sdk-core/include/aws/core/SDKConfig.h
@@ -14,4 +14,3 @@
   */
 
 #define USE_AWS_MEMORY_MANAGEMENT
-
